src/intersection/first-solution/intersection.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Creates an array of values that are in both the first and the second arrays.
+ *
+ * @param {Array} [firstArr=[]] - First array to be compared for repeated values.
+ * @param {Array} [secondArr=[]] - Second array to be compared for repeated values.
+ * @returns {Array} Returns the new array with only intersected values.
+ */
+
+export const arrayIntersection = (firstArr = [], secondArr = []) => {
```

Comment:
If you are going to throw error if no values are passed, what is the use of initializing them?